sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala

```diff
@@ -153,6 +153,14 @@ class V2SessionCatalog(sessionState: SessionState) extends TableCatalog {
     }
   }
 
+  override def renameTable(oldIdent: Identifier, newIdent: Identifier): Unit = {
```

Comment:
Can you also add test cases for v2 session catalog?

Reply:
Yeah, it'd be nice to see tests for renaming across namespaces (failing), etc.

Reply:
I just added the tests.